lib/rspec/core/configuration.rb

```diff
@@ -202,10 +202,31 @@ def only_failures_but_not_configured?
         only_failures? && !example_status_persistence_file_path
       end
 
-      # @macro add_setting
+      # @macro define_reader
       # If specified, indicates the number of failures required before cleaning
-      # up and exit (default: `nil`).
-      add_setting :fail_fast
+      # up and exit (default: `nil`). Can also be `true` to fail on exit on first
+      # failure
+      define_reader :fail_fast
+
+      # @see fail_fast
+      def fail_fast=(value)
+        case value
+        when true, 'true'
```

Comment:
Let me know if you wanted to cover more truthy values. I figured this will cover most use-cases.

Reply:
No this is fine :)